HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1744,7 +1745,7 @@ def self.get_cool_capacity_ratios(hvac_system)
   # @param heating_system [TODO] TODO
   # @param use_cop [TODO] TODO
   # @return [nil]
-  def self.set_heat_curves_central_air_source(heating_system, use_cop = false)
+  def self.set_heat_curves_dx(heating_system, use_cop = false)
```

Comment:
This method is not only called with central air source systems, renamed here. Let me know if you have a better name suggestion.

Reply:
I agree that it doesn't have to be central, but I think the "air source" part of it is important to distinguish from ground/water source systems.